add introduction of --disable-consumer-sync in konnect KIC section (#7898)
Kong/docs.konghq.com: app/konnect/gateway-manager/kic.md

**Summary** | This section displays the traffic and error rate of your KIC data plane node.  
**Analytics** | Analytics data for the KIC data plane node. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/). 

> ### Optimize performance of synchronizing configuration

Reviewer: ```suggestion
{% if_version gte:3.3.x %}
### Optimize performance of synchronizing configuration
```

> {{site.kic_product_name}} 3.3 and later provides a `--disable-consumer-sync` flag to disable synchronizing consumers with {{site.konnect_short_name}}. When your environment has a lot of `KongConsumer`s which can slow down your configuration synchronization with {{site.konnect_short_name}}, you can enable this flag to skip synchronizing consumers to make the synchronization faster. See [{{site.kic_product_name}} CLI arguments](/kubernetes-ingress-controller/latest/reference/cli-arguments/) for more details.

Reviewer: ```suggestion
When your environment has many `KongConsumer`s, they can slow down your configuration synchronization with {{site.konnect_short_name}}. {{site.kic_product_name}} provides a `--disable-consumer-sync` flag to disable synchronizing consumers with {{site.konnect_short_name}}.  You can enable this flag to make the synchronization faster. See [{{site.kic_product_name}} CLI arguments](/kubernetes-ingress-controller/latest/reference/cli-arguments/) for more details.
```
Rephrasing for better flow. Also removed the version mention, as the version tags should handle that.

> 

Reviewer: ```suggestion

{% endif_version %}

```

**Summary** | This section displays the traffic and error rate of your KIC data plane node.  
**Analytics** | Analytics data for the KIC data plane node. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/). 

### Optimize performance of synchronizing configuration

> When your environment has many `KongConsumer`s, they can slow down your configuration synchronization with {{site.konnect_short_name}}. {{site.kic_product_name}} provides a `--disable-consumer-sync` flag to disable synchronizing consumers with {{site.konnect_short_name}}. You can enable this flag to make the synchronization faster. See [{{site.kic_product_name}} CLI arguments](/kubernetes-ingress-controller/latest/reference/cli-arguments/) for more details.

Reviewer: ```suggestion
When your environment has many `KongConsumer`s, they can slow down your configuration synchronization with {{site.konnect_short_name}}. {{site.kic_product_name}} 3.3 and later version provide a `--disable-consumer-sync` flag to disable synchronizing consumers with {{site.konnect_short_name}}. You can enable this flag to make the synchronization faster. See [{{site.kic_product_name}} CLI arguments](/kubernetes-ingress-controller/latest/reference/cli-arguments/) for more details.
```
re-adding the mention of the version, since this isn't versioned via tags.

**Summary** | This section displays the traffic and error rate of your KIC data plane node.  
**Analytics** | Analytics data for the KIC data plane node. You can configure the analytics options using the [**Analytics tool**](/konnect/analytics/). 

> {% if_version gte:3.3.x %}

Reviewer: ```suggestion
```
I didn't realize this was a konnect doc 🤦‍♀️ . Doesn't need or accept version tags.

### Optimize performance of synchronizing configuration

When your environment has many `KongConsumer`s, they can slow down your configuration synchronization with {{site.konnect_short_name}}. {{site.kic_product_name}} provides a `--disable-consumer-sync` flag to disable synchronizing consumers with {{site.konnect_short_name}}. You can enable this flag to make the synchronization faster. See [{{site.kic_product_name}} CLI arguments](/kubernetes-ingress-controller/latest/reference/cli-arguments/) for more details.

> {% endif_version %}

Reviewer: ```suggestion
```
